ssc/cmod_pvsamv1.cpp

```diff
@@ -3506,4 +3698,30 @@ void cm_pvsamv1::inverter_size_check()
             SSC_WARNING);
 }
 
+const double Subhourly_Clipping_Matrix[21][21] =
```

Comment:
Looks like this clipping matrix is also present in lib_shared_inverter.cpp? If intentional, needs comments, if unintentional, one can be removed :)

Reply:
Removed.